### PR TITLE
chore(ui/alerts): change notificationRule to rule

### PR DESCRIPTION
**Pull request by 121watts**
I couldn't type or auto-complete `notificationRule` one more time.  With our types and usage of `alerts/notifications/rules.ts` directory structure I don't think we need to prefix all mentions of `rule` with `notification`.

### CODE REVIEW

File: ui/src/alerting/components/AlertColumnHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  justifyContent: 'space-between',
+  alignItems: 'center',
+}
+
+interface Props {
+  title: string
+  onCreate: () => void
+}
+
+const AlertsColumnHeader: FC<Props> = ({onCreate, title}) => {
```
Comment (ebb-tide) [inline ui/src/alerting/components/AlertColumnHeader.tsx]: the file name and component name don't match (one's alerts.. the other is alert.. )and that can get confusing, could they match please? :)

File: ui/src/pageLayout/containers/Nav.tsx [hunk 2]
```diff
@@ -10,7 +10,6 @@ import CloudNav from 'src/pageLayout/components/CloudNav'
 import AccountNavSubItem from 'src/pageLayout/components/AccountNavSubItem'
 import CloudExclude from 'src/shared/components/cloud/CloudExclude'
 import CloudOnly from 'src/shared/components/cloud/CloudOnly'
-import {FeatureFlag} from 'src/shared/utils/featureFlag'
```
Comment (ebb-tide) [inline ui/src/pageLayout/containers/Nav.tsx]: I think this is the line that is causing js tests to fail

File: ui/src/alerting/components/notifications/RulesColumn.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+type Props = StateProps
+
+const NotificationRulesColumn: FunctionComponent<Props> = ({rules}) => {
+  const handleClick = () => {
+    console.log('creating rules: ')
+  }
+
+  return (
+    <>
+      <AlertsColumnHeader title="Notification Rules" onCreate={handleClick} />
```
Comment (ebb-tide) [inline ui/src/alerting/components/notifications/RulesColumn.tsx]: This is helpful :)

File: ui/src/alerting/components/AlertColumnHeader.tsx (hunk 1 above)
Comment (121watts) [reply ui/src/alerting/components/AlertColumnHeader.tsx]: yaaas!

Comment (121watts) [reply ui/src/alerting/components/AlertColumnHeader.tsx]: good catch

**Pull request merged**